app/src/main/java/com/es/findsoccerplayers/ActivityInfoBookedMatch.java

```diff
@@ -16,16 +17,20 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_info_match_booked);
+
+        setContentView(R.layout.act_info_match_booked);
         TabLayout tabs = findViewById(R.id.info_match_booked_tabs);
         ViewPager vp = findViewById(R.id.info_match_booked_vp);
         Toolbar toolbar = findViewById(R.id.info_match_booked_toolbar);
         setSupportActionBar(toolbar);
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
 
+        Intent i = getIntent();
+        String matchID = i.getStringExtra("match");
```

Comment:
match mettiamolo come 
private static final String MATCH = "match";
Questo anche in tutte le altre activity e Fragment

Reply:
perchè?

Reply:
perche il prof ci tiene. L'ha detto anche a lezione che le chiavi siano definite come static final. Buona programazzione. So che è una rottura di palle, ma ci tiene